app/Resources/views/events/show.html.twig

```diff
@@ -31,7 +31,7 @@
             </h1>
         </div>
 
-        {% if event.isValid and event.start <= date() %}
+        {% if event.isValid and event.startWithTimezone <= date() %}
```

Comment:
`Event::isValid()` already checks that start is before now, so I think the 2nd part of this check can be removed. However, `Event::isValid()` doesn't take into account TZ, so the startWithTimezone should maybe be moved there?